packages/shopify-checkout/README.md

```diff
@@ -188,6 +188,51 @@ const checkout = await checkoutClient.discountRemove({
 });
 ```
 
+##### Making custom checkout queries to the Shopify Storefront API
+
+| Parameter   | [Type](#common-types) | Required? | Description                                           |
+| ----------- | --------------------- | --------- | ----------------------------------------------------- |
+| `query`     | `string`              | ✅        | A Shopify Storefront API query/mutation               |
+| `variables` | `object`              | ✅        | Variables for a Shopify Storefront API query/mutation |
+
+##### Example
+
+```js
+// Send any query/mutation to the Shopify Storefront API
+const checkout = await checkoutClient.query({
+  query: `
+    mutation checkoutShippingAddressUpdateV2($checkoutId: ID!, $shippingAddress: MailingAddressInput!) {
+      checkoutShippingAddressUpdateV2(checkoutId: $checkoutId, shippingAddress: $shippingAddress) {
+        checkout {
+          id
+          webUrl
+        }
+        checkoutUserErrors {
+          code
+          fields
+          message
+        }
+      }
+    }
+    `,
+  variables: {
+    checkoutId: checkoutData.id,
+    shippingAddress: {
+      address1: '123 smith street',
+      city: 'Smith',
+      country: 'USA',
+      firstName: 'John',
+      lastName: 'John'
+    }
+  }
+});
+```
+
+Other example mutations from Shopify docs:
+
+[checkoutshippingaddressupdatev2]: https://shopify.dev/api/storefront/2022-07/mutations/checkoutShippingAddressUpdateV2
+[checkoutcustomerassociatev2]: https://shopify.dev/api/storefront/2022-07/mutations/checkoutCustomerAssociateV2
```

Comment:
@KrisQ not quite...these links do not show up:

<img width="450" alt="Screen Shot 2022-07-12 at 12 54 42 PM" src="https://user-images.githubusercontent.com/5732000/178549499-e0ec0088-089c-4833-8eca-55f63b43b703.png">

Instead:
1. Put these at the bottom of the document, with the other reference-style links (I'd recommend using snake casing to separate the words):

```md
[shopify-mutation-checkout-shipping-address-update-v2]: https://shopify.dev/api/storefront/2022-07/mutations/checkoutShippingAddressUpdateV2
[shopify-mutation-checkout-customer-associate-v2]: https://shopify.dev/api/storefront/2022-07/mutations/checkoutCustomerAssociateV2
```

3. And then here, you can:

```suggestion
- [`checkoutShippingAddressUpdateV2`][shopify-mutation-checkout-shipping-address-update-v2]
- [`checkoutCustomerAssociateV2`][shopify-mutation-checkout-customer-associate-v2]
```